.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: CI
 
-on: [pull_request]
+on: [push, pull_request]
```

Comment:
Why is this necessary?

Reply:
To test the CI in my fork, wasn't intended to land in the PR.